repos/system_upgrade/el7toel8/actors/migratesendmail/tests/component_test.py

```diff
@@ -0,0 +1,24 @@
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import SendmailMigrationDecision
+from leapp.reporting import Report
+import os
+import shutil
+
+def test_actor_migration(current_actor_context):
+    backup = False
+    if os.path.exists('/etc/mail'):
```

Comment:
Unit tests should not be destructive to the host system. Please consider mocking or using dummy data. Modifying /etc/mail is wrong.

Reply:
As well as relying on any data on the host machine. Behaviour should be same on every machine where tests are processed.

Reply:
The behaviour is still the same, no matter of the host configuration - it even works if there is no sendmail package installed - every time it generates new sendmail configuration. I have no problem mocking the test - and of course it's much nicer approach, but please be consistent in the leap documentation, e.g. in the https://leapp.readthedocs.io/en/latest/unit-testing.html#testing-actors-that-modify-the-os I choosed the first bullet.

Reply:
@yarda Thanks for reporting that! That's really our mistake that something like that is written there.

Reply:
Reported here: https://github.com/oamg/leapp/issues/509

Reply:
Mocked tests.